README.md

```diff
@@ -127,9 +126,14 @@ Sugarizer Server Dashboard is an admin tool for teachers and deployment administ
 * Graphic and request: display graphics and report on previous data.
 
 To login to the Dashboard the first time, you will have to create an admin account using this command:
-
+        
 	sh add-admin.sh admin password http://127.0.0.1:8080/auth/signup
 
+Incase using Docker to login the Dashboard for the first time 

```

Comment:
Maybe replace `Incase using Docker to login the Dashboard for the first time` with something like `If Sugarizer Server was deployed using Docker Compose, use these commands to create an admin account:`